lib/puppet/functions/stdlib/nested_values.rb

```diff
@@ -1,18 +1,17 @@
 # frozen_string_literal: true
 
 # This function will return list of Hash values, the return value will be Array
-# NOTE : This function is expecting only Hash and return value will be Array
 #
 # @example :
 # $hash = {
 #   "key1" => "value1",
 #   "key2" => { "key2.1" => "value2.1"}
 # }
-# $hash.nested_values
+# $hash.stdlib::nested_values
 #
 # Output : ["value1", "value2.1"]
```

Comment:
I think the whole line will be rendered inside of a puppet marked up code block, so you might want an additional `#` on this line??
```suggestion
# # Output : ["value1", "value2.1"]
```